services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/DigitalGreenCertificateStructureProvider.java

```diff
@@ -61,6 +73,42 @@ private DirectoryOnDisk constructArchiveToPublish(DigitalGreenCertificate dgcCon
       logger.info("Writing digital green certificate to {}/{}/{}.", dgcDirectory.getName(), languageDirectory.getName(),
           archiveToPublish.getName());
     }
+
+    dgcDirectory.addWritable(getOnboardedCountriesArchive());
+    dgcDirectory.addWritable(getRulesArchive(RuleType.Acceptance, ACCEPTANCE_RULES));
+    dgcDirectory.addWritable(getRulesArchive(RuleType.Invalidation, INVALIDATION_RULES));
```

Comment:
kind of redundant information.... 
`RuleType.Acceptance, ACCEPTANCE_RULES`
`RuleType.Invalidation, INVALIDATION_RULES` 
why isn't one or the other sufficient?

Reply:
One is the name of the archive which from the techspec should be "acceptance-rules" or "invalidation-rules" and the other is the enum for which the filtering of rules is made. I could let only the enum but then I have to treat the archive name condition in the method..